crates/holochain/src/core/sys_validate/tests.rs

```diff
@@ -610,30 +612,48 @@ async fn test_agent_update() {
     let head = chain.chain_head().unwrap().unwrap();
 
     let a1 = Action::AgentValidationPkg(AgentValidationPkg {
-        author: agent2.clone(),
+        author: agents[0].clone(),
         timestamp: (head.timestamp + sec).unwrap(),
         action_seq: head.seq + 1,
         prev_action: head.action,
         membrane_proof: None,
     });
 
     let a2 = Action::Create(Create {
-        author: agent2.clone(),
+        author: agents[0].clone(),
```

Comment:
agents[0] (source chain author) creates another agent, namely agents[1]; seems to be fine too